hera_cal/lstbin.py

```diff
@@ -483,6 +483,7 @@ def lst_bin_arg_parser():
     a.add_argument('data_files', nargs='*', type=str, help="quotation-bounded, space-delimited, glob-parsable search strings to time-contiguous nightly data files")
     a.add_argument("--dlst", type=float, default=None, help="LST grid bin width")
     a.add_argument("--lst_start", type=float, default=0, help="starting LST for binner as it sweeps across 2pi LST")
+    a.add_argunent("--fixed_start", action='store_true', default=False, help="If True, make the start of the LST grid equal to lst_start, rather than the LST of the first data record.")
```

Comment:
I think `fixed_start` is a confusing parameter name that doesn't really say what it means. I think instead it should be something like `no_lst_trimming` would be much clearer.

Reply:
What about `fixed_lst_start`. There is a separate function in `hera_pspec` called `trim_lst`, so I'd prefer to stay away from that specific name..